src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php

```diff
@@ -752,9 +752,6 @@ private function getExpressionResolvers(): array
 						)
 					);
 				},
-				'uniqueValues' => static function (Scope $scope, Arg $value): array {
-					return self::createIsNonEmptyArrayAndSomethingExprPair('uniqueValues', [$value]);
```

Comment:
instead of deleting you might use `$scope->getType($value)->isNonEmptyArray()->yes()`?

Reply:
in this scope this is not relevant IMO. we don't overwrite types, so if it's non-empty-array already it stays non-empty-array without explicitly doing anything in this extension